package.json

```diff
@@ -10,19 +10,24 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@types/mocha": "^2.2.38",
-    "@types/node": "^7.0.4",
-    "@types/request": "0.0.40",
+    "@types/mocha": "^2.2.39",
+    "@types/node": "^7.0.5",
+    "@types/request": "0.0.43",
     "@types/request-promise": "^4.1.33",
     "coveralls": "^2.13.1",
     "lerna": "^2.0.0-rc.4",
     "mocha": "^3.2.0",
     "nyc": "^10.3.2",
     "request": "^2.79.0",
     "request-promise": "^4.1.1",
-    "ts-node": "^2.0.0",
+    "ts-node": "^3.0.4",
+    "tsconfig-paths": "^2.1.1",
     "tslint": "^5.1.0",
-    "typescript": "^2.1.5"
+    "typescript": "^2.2.1"
+  },
+  "dependencies": {
```

Comment:
Dependencies must be specified in individual package files, never at the top level. The modules specified in top-level `package.json` won't be installed when users install individual package like `npm install @loopback/juggler`.